prompt_manager/zero_shot.py

```diff
@@ -0,0 +1,80 @@
+from models.models import Message, ConversationRolesEnum, ConversationRolesInternalEnum, LLMType
+from prompt_manager.ipromptmanager import IPromptManager
+
+CODE_GENERATION_PROMPT = """You are a data engineer, to help retrieve data by writing python code based on a request. In this interaction, you are providing data scientist with code snippets to analyze the dataset provided.
```

Comment:
Shouldn't this prompt be set outside of this scope. It seems to me that it is very simmilar to the prompt in few_shot.py. Also if we will add to the prompt some goals, imo another reason why it should be separated from ipromptmanager.py. And btw the name alone suggest it manages prompt, not it is a PROMPT.

Reply:
For now I would stay with prompt manager maintaining and preparing conversation including storing prompts.